### DISCUSSION TITLE
AppFlowy Editor, a highly customizable rich-text editor for Flutter, is published on pub.dev

### DISCUSSION THREAD

**Discussion by annieappflowy** [AppFlowy-IO/AppFlowy#872] [Show and tell]
I can't wait to share this with you here - AppFlowy Editor is published on pub.dev https://pub.dev/packages/appflowy_editor
Calling all Flutter devs in our community to test it out and provide feedback 🙏
![image](https://user-images.githubusercontent.com/12026239/185409357-d4436869-363d-41b2-909d-c091ea6e4ea0.png)

**Comment by aleripe** [comment]
Any plans of expanding this editor to mobile? :)

**Reply by annieappflowy** [reply]
Thanks for asking. Mobile is scheduled to start in Q1, and we aim to release the first version by Q1 or early Q2.
Just published our new roadmap: https://github.com/AppFlowy-IO/AppFlowy/discussions/1715
Look forward to getting your feedback!

**Reply by aleripe** [reply]
This is really great news! Keep up the good job! :)
